ocaml-lsp-server/test/e2e/__tests__/textDocument-formatting.test.ts

```diff
@@ -108,13 +110,15 @@ describe("textDocument/formatting", () => {
 
     expect(result).toMatchObject([
       {
-        newText:
-          ["module Test : sig",
-            "  type t =",
-            "    | Foo",
-            "    | Bar",
-            "    | Baz",
-            "end", ""].join(newline)
+        newText: [
+          "module Test : sig",
+          "  type t =",
+          "    | Foo",
+          "    | Bar",
+          "    | Baz",
+          "end",
+          "",
+        ].join(newline),
```

Comment:
Why do we need these unrelated changes? Perhaps they can be in a separate PR?

Reply:
that was because of running `make fmt`. I will exclude this change.

Reply:
Yes it because of the recent PR that changed the ocamlformat tests? If so we can revert those changes.

Reply:
I reverterd those changes in master. So if you rebase, you shouldn't need these unrelated changes anymore.